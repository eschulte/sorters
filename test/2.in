87686876 9879789 4756456 4534657 45354 9878 123 2 1
