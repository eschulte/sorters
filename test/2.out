1 2 123 9878 45354 4534657 4756456 9879789 87686876 
